garb/garb_main.cpp

```diff
@@ -113,10 +112,7 @@ extern "C" {
   }
 }
 #endif
-||||||| b6cd015f
-=======
-#include <signal.h> // sigaction
```

Comment:
Please move the inclusion of `signal.h` in line number 20 outside the `#if defined(WITH_COREDUMPER)` block.

This causes compilation failure when just run with scons.
```
garb/garb_main.cpp: In function 'int garb::main(int, char**)':
garb/garb_main.cpp:213:26: error: aggregate 'garb::main(int, char**)::sigaction isa' has incomplete type and cannot be defined
  213 |         struct sigaction isa;
      |                          ^~~
garb/garb_main.cpp:215:26: error: 'SIG_IGN' was not declared in this scope
  215 |         isa.sa_handler = SIG_IGN;
      |                          ^~~~~~~
garb/garb_main.cpp:217:24: error: 'SIGPIPE' was not declared in this scope
  217 |         if (sigaction (SIGPIPE, &isa, NULL))

```

Reply:
done